.editorconfig

```diff
@@ -1,8 +1,8 @@
 root = true
 
 [*]
-indent_style = space
-indent_size = 2
+indent_style = tab
+indent_size = 4
```

Comment:
I agree. I've become a fan of 2 space indents. Just because more JS teams do this : https://basarat.gitbooks.io/typescript/content/docs/styleguide/styleguide.html#spaces and its purely consensus vs. *what might be the better thing* (even though as C# dev I used to love 4 space indents) :rose: